### PR TITLE
fix: multiple usability fixes

### PR DESCRIPTION
**Pull request by carlos-affinidi**
- Fixed state issues. Clearing state on every session change event.
- When initializing the app open the walkthrough once
- Add error & information messages to CSV issuance
- Add JSON Schema format specifications link to snippets
- Moved Generate App step before Schema Creation on Walkthorugh
- Default walletUrl to "http://localhost:3000/holder/claim"

### CODE REVIEW

File: src/features/issuance/csvCreationService.ts [hunk 1]
```diff
@@ -94,18 +94,20 @@ const uploadCsvFile = async (input: { schema: Schema; projectId: string }) => {
...
   } catch (error: unknown) {
     const parsedCsvUploadError = parseUploadError(error)
 
     if (parsedCsvUploadError) {
-      ext.outputChannel.appendLine(
-        l10n.t(
-          `${csvMessage.csvValidationError} {0} ${JSON.stringify(parsedCsvUploadError, null, 2)}`,
-        ),
+      window.showErrorMessage(
+        l10n.t(`${csvMessage.csvValidationError} Check Affinidi Output for more details.`),
```
Comment (OleksiiO-affinidi) [inline src/features/issuance/csvCreationService.ts:107]: move text to `messages.ts` pls

**Review by anusha-kommu — APPROVED** [review]

File: src/auth/authentication-provider/affinidi-authentication-provider.ts [hunk 2]
```diff
@@ -159,10 +160,10 @@ export class AffinidiAuthenticationProvider implements AuthenticationProvider, D
       },
     })
 
-    this.handleRemoveSession()
+    await this.handleRemoveSession()
   }
 
-  handleRemoveSession = (): void => {
+  handleRemoveSession = async (): Promise<void> => {
```
[the remark this answers is not in the document]
Comment (carlos-affinidi) [reply src/auth/authentication-provider/affinidi-authentication-provider.ts:166]: Oh I missed this. It was a previous attempt to clear state. Will fix in a follow up PR.

File: src/snippets/send-vc-offer-to-email/javascript.ts [hunk 3]
```diff
@@ -31,8 +31,8 @@ const issuanceResponse = await fetch('${input.issuanceApiUrl}/issuances', {
         // claim link will be sent by email
         method: 'email',
       },
-      // set walletUrl if you have implemented your own wallet with claim flow support
-      // walletUrl: "https://my.wallet.com/claim",
+      // set walletUrl to your wallet URL with claim flow support
+      walletUrl: "http://localhost:3000/holder/claim",
```
[the remark this answers is not in the document]
Comment (carlos-affinidi) [reply src/snippets/send-vc-offer-to-email/javascript.ts:35]: True, but now that we have the reference app we can start sunsetting wallet.affinidi.com. We can push for devs to create their wallets instead.

[the remark this answers is not in the document]
Comment (carlos-affinidi) [reply src/snippets/send-vc-offer-to-email/javascript.ts:35]: Main purpose of this change is that people that have created their reference app don't have to change this value. Its to simplify onboarding flow

**Pull request merged**